league/unit.py

```diff
@@ -0,0 +1,45 @@
+import pygame
+from typing import List
+from team import Team
+from ability import Ability
+
+class Unit:
+    def __init__(self, base_health: int, base_damage: int, base_armor: int, base_range: int, base_speed: int, abilities: List[Ability], team: Team, enemy_team: Team) -> None:
+        self._base_health = base_health
+        self._base_damage = base_damage
+        self._base_armor = base_armor
+        self._base_range = base_range
+        self._base_speed = base_speed
+        self._level = 1
+        self._team = team
+        self._enemy_team = enemy_team
+
+    def apply_debuff(self, debuff: Ability):
+        "Take an Ability object and stores the debuff"
+        pass
+
+    def implement_debuffs(self):
+        "Applies the debuffs onto the unit"
+        pass
+
+    def update(self):
+        "Updates the player debuffs when the duration of debuff is over"
+        pass
+
+    def get_damage(self) -> int:
+        return self._base_damage
```

Comment:
With the change to having a Stats() object, we need to `return self._base_stats.damage`